x-pack/plugins/cloud_security_posture/public/components/no_findings_states.tsx

```diff
@@ -125,7 +125,7 @@ const IndexTimeout = () => (
       <p>
         <FormattedMessage
           id="xpack.csp.noFindingsStates.indexTimeout.indexTimeoutDescription"
-          defaultMessage="Data should appear in less than 10 minutes after elastic-agent is successfully deployed. {docs}"
+          defaultMessage="Collecting findings is taking longer than expected, please review our {docs} or reach out to support"
```

Comment:
@tehilashn should we add 'reach out to support' or remove that/add something else?

The fact it is taking longer than expected not necessarily means there's an issue they could not trouble shoot by themselves, right?

Reply:
based on the last comment from [tin](https://github.com/elastic/kibana/issues/154292#issuecomment-1499400546), I think it should be something like:
Collecting findings is taking longer than expected. [Learn more](https://www.elastic.co/guide/en/security/current/cspm-security-posture-faq.html).